tests/cases/fourslash/indentationInJsx3.ts

```diff
@@ -4,28 +4,20 @@
 ////function foo () {
 ////   return (
 ////        <div>
-////hello
-////goodbye
+/////*0*/hello
+/////*1*/goodbye
 ////        </div>
 ////    )
 ////}
 
-goTo.position(21);
-verify.textAtCaretIs("return");
-goTo.position(38);
-verify.textAtCaretIs("<div>");
-goTo.position(44);
-verify.textAtCaretIs("hello");
-goTo.position(50);
-verify.textAtCaretIs("goodbye");
+goTo.marker('0');

```

Comment:
I think this whole test would be simpler with `verify.currentFileContentIs` -- and you could just remove everything except the `<div>` from the file.